### PR TITLE
Support building images and videos

### PR DESCRIPTION
**Pull request by moriaam**

### CODE REVIEW

File: packages/rebox/web/src/build-web/get-config.ts [hunk 1]
```diff
@@ -65,6 +65,16 @@ export const getConfig = (entryPointPath: string, outputPath: string, htmlTempla
           },
         ],
       },
+      {
+        test: /\.png$/,
+        exclude: /node_modules/,
+        loader: 'file-loader?name=images/[name].[ext]',
```
Comment (deepsweet) [inline packages/rebox/web/src/build-web/get-config.ts]: let's make it `images/[name].[hash].[ext]` like the `js/[name].[chunkhash].js` above

File: packages/rebox/web/src/build-web/get-config.ts [hunk 2]
```diff
@@ -65,6 +65,16 @@ export const getConfig = (entryPointPath: string, outputPath: string, htmlTempla
...
+      },
+      {
+        test: /\.mp4$/,
+        exclude: /node_modules/,
+        loader: 'file-loader?name=videos/[name].[ext]',
```
Comment (deepsweet) [inline packages/rebox/web/src/build-web/get-config.ts]: `file-loader` should be an explicit dependency of Rebox.

Also, would be nice to do the same for the rest of methods like `runWeb` and native ones.

Comment (moriaam) [reply packages/rebox/web/src/build-web/get-config.ts]: I agree with you it makes to have it for native as well, but honestly I have no idea what's going on there :/
can you help me out with adding the native ones?

Comment (deepsweet) [reply packages/rebox/web/src/build-web/get-config.ts]: sure :)

Comment (moriaam) [reply packages/rebox/web/src/build-web/get-config.ts]: thanks! 🙏

**Pull request merged**